src/utils.ts

```diff
@@ -1981,14 +2066,19 @@ export class newTooltip {
 
   private buildText(context: CanvasRenderingContext2D): [string[], Vertex] {
     context.save();
-    context.font = `${this.fontsize}px sans-serif`
-    let printedRows = ['Information: '];
+    context.font = `${this.fontsize}px sans-serif`;
+    let printedRows = [];
     let textLength = context.measureText(printedRows[0]).width;
     this.dataToPrint.forEach((value, key) => {
-      const text = `  - ${key}: ${this.formatValue(value)}`;
+      let text: string;
```

Comment:
Should probably set as null or empty (beware of further non falsy check) since you check it afterwards

Reply:
Done